### PR TITLE
ansible: add list groups in an inventory example

### PR DESCRIPTION
**Pull request by aherst**
Add example to list groups in an inventory.

- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/ansible.md [hunk 1]
```diff
@@ -27,3 +27,7 @@
 - Execute a command using a custom inventory file:
 
 `ansible {{group}} -i {{inventory_file}} -m command -a '{{my_command}}'`
+
+- List the groups in an inventory:
+
+`ansible localhost -m debug -a 'var=groups.keys()'`
```
Comment (bl-ue) [inline pages/common/ansible.md:33]:
```suggestion
`ansible localhost -m debug -a '{{var=groups.keys()}}'`
```

Comment (Waples) [reply pages/common/ansible.md:33]: If the change suggestion from @bl-ue is done, then it's good to go.

**Review by Waples — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Comment by aherst** [comment]
Is there anything left for me to do to have this merged?

**Comment by bl-ue** [comment]
Nope, done. :+

**Pull request merged**